docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -184,27 +188,27 @@ Here you can set the maximum transmission unit (MTU) of the VPN interface. The r
 
 #### Restore VPN automatically
 
-If enabled, this feature automatically re-enables AdGuard’s local VPN after it has been turned off due to network absence, tethering, or low-power mode.
+This option automatically re-enables AdGuard’s local VPN after it has been turned off due to network absence, tethering, or low-power mode.
 
 #### Packet capture (PCAP)
 
-If enabled, AdGuard will create the special file name `timestamp.pcap` (for instance, 1682599851461.pcap). It contains all network packets transferred through the VPN. This file is located in the app cache directory and can be analyzed with the Wireshark program.
+This option makes AdGuard create a file `timestamp.pcap` (for instance, 1682599851461.pcap) that lists all network packets transferred through the VPN and is located in the app cache directory. This file can be analyzed with the Wireshark program.
 
 #### Include Wi-Fi gateway in VPN routes
 
-If you want the gateway IP addresses to be added to VPN routes when on Wi-Fi, enable this feature.
+This option adds the gateway IP addresses to VPN routes when on Wi-Fi.
 
 #### IPv4 address
 
 Here you can enter the IP address that will be used to create a TUN interface. By default, it is `172.18.11.218`.
 
 #### Forcibly route LAN IPv4
 
-When enabled, AdGuard filters all LAN connections, including local IPv4 network traffic, even if the *Route all LAN IPv4 connections* option is enabled.
+This option makes AdGuard filter all LAN connections, including local IPv4 network traffic, even if the *Route all LAN IPv4 connections* option is enabled.
 
 #### Route all LAN IPv4 connections
 
-Once enabled, AdGuard excludes LAN connections from filtering for simple networks. May not work for complex networks. Works only with the *Forcibly route LAN IPv4* option disabled.
+This option makes AdGuard exclude LAN connections from filtering for simple networks. May not work for complex networks. Works only with the *Forcibly route LAN IPv4* option disabled.
```

Comment:
If this is enabled, AdGuard will exclude LAN connections from filtering for simple networks. This may not work for complex networks. This setting only applies if *Forcibly route LAN IPv4* is disabled.